### DISCUSSION TITLE
Deleting the Workflow

### DISCUSSION THREAD

**Discussion by mursaleen-bkk** [microsoft/farmvibes-ai#57] [Q&A] [answered]
Hello!
Sir I am working on farmvibes-ai. I run 3 workflows which take huge time and all are failed. so then I stop my farmvibes-ai cluster and again start it. After that I run an another workflow but is show the status is queue. And there is no workflow running. so how can i trouble shoot this.....

**Comment by renatolfc** [comment]
Hello @mursaleen-bkk, have you tried using the [`cancel_run`](https://github.com/microsoft/farmvibes-ai/blob/main/src/vibe_core/vibe_core/client.py#L123) method in the client?

That should  eventually remove all workflows from the orchestrator's queue.

**Comment by brsilvarec** [comment, marked as answer]
Hi @mursaleen-bkk,

   Thanks for using our project.  I am assuming you are trying to cancel a workflow. If this is the case, you can try to use the function `run. cancel()`  or `client.cancel_run(run_id)`. 

Please, can you provide the screenshot or error log of failed workflow? Perharps, we can help you with this as well.

**Reply by mursaleen-bkk** [reply]
I would like to thank you for your reply it was very helpful for me. Another thing I want to discuss that my workflows were failed due to less storage of my server.
